THRESHOLD_GYROSCOPE: [0.03, 0.03, 1.9]
THRESHOLD_COORDINATES: 0.047
